pcsx2-qt/Settings/GraphicsSettingsWidget.ui

```diff
@@ -55,7 +55,7 @@
    <item>
     <widget class="QTabWidget" name="tabs">
      <property name="currentIndex">
-      <number>0</number>
+      <number>7</number>
```

Comment:
Don't forget to revert this back to 0 prior merging

Reply:
Done